docs/src/pages/components/data-grid/rendering/rendering.md

```diff
@@ -155,7 +155,7 @@ const columns: Columns = [
 The `ColDef` type has properties to apply class names and custom CSS on the cells.
 
 - `cellClassName`: to apply class names on every cell. It can also be a function.
-- `align`: to align the content of the cells. It must be 'left' | 'right' | 'center'.
+- `align`: to align the content of the cells. It must be 'left' | 'right' | 'center'. (Note you must use `headerAlign` to align the content of the header.)
```

Comment:
Maybe we could just add another row?

`align`: to align the content of the cells. It must be 'left' | 'right' | 'center'
`headerAlign`: to align the column header cells. It must be 'left' | 'right' | 'center'

Reply:
headerAlign is already documented after